HotSpot/Models/Booking.m

```diff
@@ -10,6 +10,7 @@
 
 #import "Parse.h"
 #import "Listing.h"
+#import "TimeInterval.h"
```

Comment:
did you already import Booking.h in this file? if you import Booking.h, and Booking.h imports TimeInterval.h and Listing.h, then Booking.m does not have to re-import those.